ambient.d.ts

```diff
@@ -559,3 +562,6 @@ declare module '@mojaloop/central-services-stream' {
 declare module '@hapi/good'
 declare module 'hapi-openapi'
 declare module 'blipp'
+declare module 'canonical-json' {
+  export default function stringify(value: any, replacer ?: ((this: any, key: string, value: any) => any) | undefined, space ?: string | number | undefined): string
```

Comment:
maybe this:
```
export default function stringify(value: unknonw, replacer ?: ((this: unknown, key: string, value: unknown) => unknown), space ?: string | number): string
```
there is no need for undefined alternative type if you are using `?` to specify an optional parameter

Reply:
I would add type for replacer function and use it here for clarity

Reply:
Thanks, good point. I'll follow up with these fixes.